src/components/graph.tsx

```diff
@@ -125,8 +226,13 @@ export function extractGraphData(data: string): GraphData | null {
   // TODO: Currently don't support arrays of arrays
   if (typeof array[0] === "object" && !(array[0] instanceof Array)) {
     // Fields that match the possible x labels and are in every object in the array
+    // x matches are either in the possible x labels or can be converted to a date
     const xMatches = Object.keys(array[0])
-      .filter((key) => checkStringMatch(key, possibleXlabels))
+      .filter(
+        (key) =>
+          checkStringMatch(key, possibleXlabels) ||
+          attemptDatetimeConversion(array.map((obj) => obj[key])) !== null
+      )
```

Comment:
I don't know why, but this doesn't fix the Chattermill example :thinking: 

This is an example of the format: `2023-08-02T14:02:16.666Z`

Can send you what you need to repro if you like?

![Screenshot from 2023-08-23 10-16-03](https://github.com/Superflows-AI/chat-ui/assets/33871096/44d85be3-e7c0-4e76-b103-628f7cec7046)